Ukulele/code.py

```diff
@@ -71,56 +75,520 @@
+# so these maps also correct for that
+
+
+# Bottom up along both sides at once
+pixel_map_reverse = PixelMap(
+    pixels,
+    [
+        0,
+        103,
+        1,
+        102,
+        2,
+        101,
+        3,
+        100,
+        4,
+        99,
+        5,
+        98,
+        6,
+        97,
+        7,
+        96,
+        8,
+        95,
+        9,
+        94,
+        10,
+        93,
+        11,
+        92,
+        12,
+        91,
+        13,
+        90,
+        14,
+        89,
+        15,
+        88,
+        16,
+        87,
+        17,
+        86,
+        18,
+        85,
+        19,
+        84,
+        20,
+        83,
+        21,
+        82,
+        22,
+        81,
+        23,
+        80,
+        24,
+        79,
+        25,
+        78,
+        26,
+        77,
+        27,
+        76,
+        28,
+        75,
+        29,
+        74,
+        30,
+        73,
+        31,
+        72,
+        32,
+        71,
+        33,
+        70,
+        34,
+        69,
+        35,
+        68,
+        36,
+        67,
+        37,
+        66,
+        38,
+        65,
+        39,
+        64,
+        40,
+        63,
+        41,
+        62,
+        42,
+        61,
+        43,
+        60,
+        44,
+        59,
+        45,
+        58,
+        46,
+        57,
+        47,
+        56,
+        48,
+        55,
+        49,
+        54,
+        50,
+        53,
+        51,
+        52,
+    ],
+    individual_pixels=True,
+)
+
+# Starts at the bottom and goes around clockwise
+pixel_map_around = PixelMap(
+    pixels,
+    [
+        0,
+        1,
+        2,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        9,
+        10,
+        11,
+        12,
+        13,
+        14,
+        15,
+        16,
+        17,
+        18,
+        19,
+        20,
+        21,
+        22,
+        23,
+        24,
+        25,
+        26,
+        27,
+        75,
+        74,
+        73,
+        72,
+        71,
+        70,
+        69,
+        68,
+        67,
+        66,
+        65,
+        64,
+        63,
+        62,
+        61,
+        60,
+        59,
+        58,
+        57,
+        56,
+        55,
+        54,
+        53,
+        52,
+        51,
+        50,
+        49,
+        48,
+        47,
+        46,
+        45,
+        44,
+        43,
+        42,
+        41,
+        40,
+        39,
+        38,
+        37,
+        36,
+        35,
+        34,
+        33,
+        32,
+        31,
+        30,
+        29,
+        28,
+        76,
+        77,
+        78,
+        79,
+        80,
+        81,
+        82,
+        83,
+        84,
+        85,
+        86,
+        87,
+        88,
+        89,
+        90,
+        91,
+        92,
+        93,
+        94,
+        95,
+        96,
+        97,
+        98,
+        99,
+        100,
+        101,
+        102,
+        103,
+    ],
```

Comment:
Please revert this change. Then disable Black for this section, and reenable it after the end of the section.